### DISCUSSION TITLE
Structure of reading routines

### DISCUSSION THREAD

**Discussion by gcanasherrera** [euclidlib/euclidlib#18] [General]
Where should new reading routines be added? Same `.py`? One `.py` for each probe (i.e: pk_wl, 2pcf_wl)?

Should we store auxiliary functions in a different common place (aka, new `.py`)?

**Comment by ntessore** [comment]
I thought that one module per "responsible party" (e.g., PK-WL, 2PCF-WL, PHZ, etc.) might be a good idea, so we could set up CODEOWNERS in the future, and have changes reviewed by the people responsible for producing the official products.

**Comment by gcanasherrera** [comment]
This is now the case, closing!
